app/metrics.yaml

```diff
@@ -23,3 +23,152 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+pages:
+  page_load:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long each page takes to load.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+url:
+  domains:
+    type: counter
+    lifetime: application
+    description: >
+      Counting how many domains are visited in a session.
```

Comment:
What is a session? Are the domains unique?

Reply:
We use a hashSet to identify different domains, but we are not interested in sending domain names to the data server. We only send hashSet().size().